Library/Formula/libghthash.rb

```diff
@@ -0,0 +1,62 @@
+class Libghthash < Formula
+  homepage 'http://www.bth.se/people/ska/sim_home/libghthash.html'
```

Comment:
Use double-quotes everywhere (and generally run `brew audit --strict` for new formulae and fix all the warnings)